### ISSUE TITLE
[BUG] Connection is failing

### ISSUE THREAD

**Issue by stinovlas** [etianen/django-python3-ldap#245]
With `LDAP_AUTH_USE_TLS=True`, LDAP connection is failing with `django-python3-ldap==0.15.0`:

```
File "/app/venv/lib/python3.8/site-packages/django_python3_ldap/ldap.py", line 182, in connection
    c = ldap3.Connection(
TypeError: __init__() got an unexpected keyword argument 'tls'
```
It seems to be originating here: https://github.com/etianen/django-python3-ldap/blob/master/django_python3_ldap/ldap.py#L178

When I examined `ldap3` library I indeed didn't find any `tls` argument in `ldap3.Connection`. It's quite weird that it passed the tests at all.

**Comment by FlipperPA** [comment]
@stinovlas @etianen This should be fixed here:

https://github.com/etianen/django-python3-ldap/pull/246

If you could verify, that would be fantastic.

**Comment by stinovlas** [comment]
@FlipperPA The fix in this branch works for my usecase :+1:. I also checked the code diff, LGTM :slightly_smiling_face:.

**Comment by jrbeilke** [comment]
+1 the fix in #246 resolved the tls connection issue for me

@etianen could we get this fix merged and a new release?

**Comment by etianen** [comment]
Merged and released as 0.15.1.

Thanks for the quick spot and bugfix everyone!
